app/docs/[slug]/page.tsx

```
@@ -0,0 +1,165 @@
+								<p className="text-xl text-muted-foreground">
+									{page.description}
+								</p>
+							)}
+						</div>
+					</div>
+
+					{/* Page content */}
+					<div className="mt-12">
+						{page.body ? (
+							<div
+								dangerouslySetInnerHTML={{__html: page.body}}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)